test/wrapper_test.py

```diff
@@ -874,11 +870,14 @@ def _mime_vs_type(mime, requested_type):
         self.assertEqual(0, _mime_vs_type("text/n3", N3))
         self.assertEqual(0, _mime_vs_type("text/turtle", TURTLE))
         self.assertEqual(0, _mime_vs_type("application/turtle", TURTLE))
-        self.assertEqual(0, _mime_vs_type("application/ld+json", JSON))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/ld+json", JSONLD))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", XML))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", RDF))  # Warning
-        self.assertEqual(0, _mime_vs_type("application/rdf+xml", RDFXML))  # Warning
+        self.assertEqual(0, _mime_vs_type("application/json", JSON))  # Warning
+
+        # graph.load() is deprecated, it will be removed in rdflib 6.0.0. Please use graph.parse() instead.
```

Comment:
I don't understand why this issue is seen as I can't find use of `load()` in the SPARQLWrapper code base! Only use, correctly, of `parse()`.

Reply:
Oh, I see the comment above about `load()` being replaces with `parse()`. I must be looking at older test results.